### ISSUE TITLE
is g711_ulaw input format supported?

### ISSUE THREAD

**Issue by da1z** [openai/openai-realtime-api-beta#18]
is g711_ulaw formatted supported? I followed twillio example and sending this session updat:

ts```
  const sessionUpdate = {
        type: 'session.update',
        session: {
          turn_detection: { type: 'server_vad' },
          input_audio_format: 'g711_ulaw',
          output_audio_format: 'g711_ulaw',
          voice: VOICE,
          instructions: SYSTEM_MESSAGE,
          modalities: ['text', 'audio'],
          temperature: 0.8,
        },
      };
```

but response looks like this:
ts```
Session updated successfully: {
......
  session: {
.......
    input_audio_format: 'pcm16', <= NOT UPDATED
    output_audio_format: 'g711_ulaw',
    input_audio_transcription: null,
    tool_choice: 'auto',
    temperature: 0.8,
    max_response_output_tokens: 'inf',
    tools: []
  }
}

```

**Comment by janak2** [comment]
Having the same issue.

**Comment by janak2** [comment]
So, I tested it with both pcm16 and g711_ulaw input audio and the input_audio_format does get updated but the session update response returns incorrect input audio format.

**Comment by themarcosf** [comment]
Same issue here. Can anyone send the URL of the Twilio documentation you are using?

**Comment by mrkww** [comment]
> Thanks folks. Would be helpful if we could get some `session.id`s from the `session.created` event!

sess_AFxDpCBHVOU5AFv3j2Nr1
Updated voice to 'shimmer' and input_audio_format to 'g711_alaw'
session.updated event reports input_audio_format is 'pcm16'
I am sending g711_alaw audio successfully and receive responses

Hope this helps!

**Comment by themarcosf** [comment]
I am setting output_audio_format to 'g711_ulaw' and it returns an Int16 array as in pcm

**Comment by da1z** [comment]
> Thanks folks. Would be helpful if we could get some `session.id`s from the `session.created` event!

@khorwood-openai sess_AG4Egp7ID6a5ibPQ7x39w

**Comment by K-Mistele** [comment]
See my comment on #8: https://github.com/openai/openai-realtime-api-beta/issues/8#issuecomment-2411857042

TL;DR the API supports G.711 mu-law 8khz, but the client does not.
